src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map(property => {
+      return property
+        .split(':')
+        .map(element => element.trim())
+        .filter(element => element !== '');
```

Comment:
The filter(Boolean) method is commonly used in JavaScript to remove falsy values from an array. When filter method is called with Boolean as the condition, the filter method will remove any falsy values (such as your "") from the array.

To better visualize this example, `filter(Boolean)` is equal to `filter(element => Boolean(element))` and Boolean("") evaluates to false in your case
It's more like a trick, but it's good to know it :D